File: src/main/java/com/koddy/server/coffeechat/domain/repository/query/MentorReservedScheduleQueryRepository.java
package com.koddy.server.coffeechat.domain.repository.query;

import com.koddy.server.coffeechat.domain.model.CoffeeChat;

import java.util.List;

public interface MentorReservedScheduleQueryRepository {
    List<CoffeeChat> fetchReservedCoffeeChat(final long mentorId, final int year, final int month);
}

File: src/main/java/com/koddy/server/coffeechat/domain/repository/query/MentorReservedScheduleQueryRepository.kt
package com.koddy.server.coffeechat.domain.repository.query

import com.koddy.server.coffeechat.domain.model.CoffeeChat

interface MentorReservedScheduleQueryRepository {
    fun fetchReservedCoffeeChat(
        mentorId: Long,
        year: Int,
        month: Int,
    ): List<CoffeeChat>
}
